external/aws/output.tf

```diff
@@ -1,5 +1,5 @@
 output "ubuntu-20-04-latest-name" {
-  value = data.aws_ami.ubuntu.description
+  value = data.aws_ami.ubuntu
```

Comment:
This was changed to `value = data.aws_ami.ubuntu.description` in this commit https://github.com/redteaminfra/redteam-infra/commit/7663403bf6b2f054a02aa68fa10f126bf201e117, why are we reverting it?

Reply:
Looks like whatever I had initially pulled from had what I am committing despite I never changed it. The only thing I had added was 

```
output "proxy_public_ips" {
  value = [for ip in aws_instance.proxy : ip.public_ip]
}
  ```